lib/compiler/src/node.h

```diff
@@ -4,26 +4,26 @@
 #include <string>
 
 #include "tl/optional.hpp"
-#include "op-meta/op-meta.h"
+#include "op-attrs/op-attrs.h"
 
 namespace FlexFlow {
 namespace ffc {
 
 struct Node {
   Node() = delete;
-  Node(size_t guid, OperatorParameters const &op_params);
+  Node(size_t guid, PCGOperatorAttrs const &op_params);
 
   std::string to_string(void) const;
 
-  using AsTuple = std::tuple<size_t &, OperatorParameters &, tl::optional<size_t> &>;
-  using AsConstTuple = std::tuple<size_t const &, OperatorParameters const &, tl::optional<size_t> const &>;
+  using AsTuple = std::tuple<size_t &, PCGOperatorAttrs &, tl::optional<size_t> &>;
```

Comment:
`AsTuple` shouldn't be necessary anymore (actually I think this whole file shouldn't be necessary anymore)